src/App.tsx

```diff
@@ -1,35 +1,181 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import React, { useEffect, useState } from "react";
+import "./App.css";
+import Form from "./Form";
+import CardList from "./CardList";
 
-function App() {
-  const [count, setCount] = useState(0)
+interface Block {
+  id: number;
+  firstTeam: string;
+  secondTeam: string;
+  tickets: string;
+  stadium: string;
+}
+
+const App: React.FC = () => {
+  const [firstTeam, setFirstTeam] = useState<string>("");
```

Comment:
To reduce the amount of useStates and improve readability you can manage form data state like this 
```
const [formData, setFormData] = useState<Block>({
    id: 0,
    firstTeam: "",
    secondTeam: "",
    tickets: "",
    stadium: "",
  });
```
Also, you can try to find a way to group error states))